.github/workflows/release.yml

```diff
@@ -35,6 +35,18 @@ env:
   # PURE: true
   # NOARCH: true
 
+  STEP_GET_BUILD_REQUIREMENTS: |
+    SETUP_REQUIRES=$(python -c '\
+    import configparser as p
```

Comment:
How about using `setuptools.dist.Distribution` here, e.g.
```python
from setuptools.dist import Distribution
dist = Distribution()
dist.parse_config_files()
dist.setup_requires
```

Reply:
Ah nice, that is better. There will be possible env markers in the output like `package; python_version < '3.8'`, but actually that should be handled by passing to pip with `pip install -r` instead of `pip install` - that will actually evaluate the requirements and install the appropriate packages. So instead of storing an environment variable to pass to pip, this should become
```bash
python -c 'from setuptools.dist import Distribution
dist = Distribution()
dist.parse_config_files()
print("\n".join(dist.setup_requires))' | pip install -r /dev/stdin
```